### ISSUE TITLE
Remove the code that places users directly in our ask question box when starting a new chat.

### ISSUE THREAD

**Issue by MarkGOC** [dto-btn/ssc-assistant#581]
We need to remove the autofocus code that places users directly in our ask question box when starting a new chat.  As it might hurt mental model creation for some disabled users.

As part of WCAG it is not recommended to do this.  Keyboard users should have a skip nav and mouse users will have to click the box.

**Why** 
-  from review with AAACT team.
